ftplugin/jq.vim

```diff
@@ -0,0 +1 @@
+setlocal commentstring=#%s
```

Comment:
Are comments not showing up for you? They appear to work for me. Sorry I didn't notice your PR till now :)
https://github.com/vito-c/jq.vim/blob/master/syntax/jq.vim#L56

Reply:
Comments currently do show up just fine. I suggest setting `commentstring` to something jq supports – from the man page it doesn’t support C-style comments `/* … */` which is Vim’s default but supports #-comments. Help says `commentstring` is only used for inserting manual folding markers – I don’t actually use that. But plugins – like ‘vim-commentary – use that option to format comments so it is useful to have supported value here.

Reply:
I use TComment and it supports jq so I must have missed this.